src/Fabulous.AST.Tests/UnitTest1.fs

```diff
@@ -0,0 +1,22 @@
+module Fabulous.AST.Tests
+
+open NUnit.Framework
+
+open Fabulous.AST
+open type Fabulous.AST.Node
+
+[<Test>]
+let CanCompileBasicTree () =
+    let tree =
+        ModuleOrNamespace(
+            Range("test.fs", Position(1, 2), Position(3, 4))
```

Comment:
When generating code with Fantomas, every `range` can be `(0,0), (0,0)`. The range value only matters when we do the `Trivia` processing when constructing the `Oak`. Because you are making your own Oak, this doesn't really matter. You can ignore it for your end-users.